manifests/e/EclipseAdoptium/TemurinJRE/11/11.0.12.7/EclipseAdoptium.TemurinJRE.11.locale.en-US.yaml

```diff
@@ -0,0 +1,22 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: EclipseAdoptium.TemurinJRE.11
+PackageVersion: 11.0.12.7
+PackageLocale: en-US
+LicenseUrl: https://adoptium.net/about.html
+Moniker: temurinjre11
+Tags:
+- openjdk
+- java
+- jvm
+- jre
+- temurin
+PackageUrl: https://adoptium.net
+Publisher: Eclipse Adoptium
+PackageName: Adoptium Temurin JRE with Hotspot 11
```

Comment:
```suggestion
PackageName: Eclipse Temurin JRE with Hotspot 11
```

Reply:
Vendor name has changed to `Eclipse Adoptium` instead of `Eclipse Foundation` that was one of the premises for this change.